configure.in

```diff
@@ -5,6 +5,7 @@ AC_CONFIG_MACRO_DIR([./buildutils])
 AC_CANONICAL_TARGET
 AC_CONFIG_SRCDIR([include/exc_platform.h])
 AC_CONFIG_HEADER([include/config.h Driver/atool/src/include/nata/nata_config.h])
+m4_include([./buildutils/m4_ax_compiler_vendor.m4])
```

Comment:
I don't really know the m4 programming language so I can only trust you on this even though the syntax seems to be fine. Rest is ok!

Reply:
I don't know it either. Copied it from internet examples.